cmd/generate/Dockerfile.template

```diff
@@ -1,20 +1,33 @@
 # DO NOT EDIT! Generated Dockerfile for {{.main}}.
-FROM {{.builder}} as builder
+ARG GO_BUILDER={{.builder}}
+ARG UBI8_MINIMAL=registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Wondering if we want to decouple the version from the arg name with `UBI_MINIMAL` or even simply `GO_RUNTIME`